README.md

```diff
@@ -149,21 +165,21 @@ import com.yoti.api.client.ProfileException;
 try {
     final ActivityDetails activityDetails = client.getActivityDetails(token);
     final HumanProfile profile = activityDetails.getUserProfile();
-    //use the profile to extract attributes.
+    //use the profile to extract attributes
 } catch (ProfileException e) {
     LOG.info("Could not get profile", e);
-    // do something meaningful.
+    // do something meaningful
 }
 ```
 
-**Important: encrypted tokens are intended for single use only. You must not invoke the client using the same token more than once.**
+`com.yoti.api.client.ActivityDetails` gives you access to the `com.yoti.api.client.ApplicationProfile` of your app as well as the `com.yoti.api.client.HumanProfile` of the user that has shared with you. 
```

Comment:
you can remove the ApplicationProfile bit

Reply:
ok